src/main/java/nextstep/subway/domain/Line.java

```diff
@@ -0,0 +1,74 @@
+package nextstep.subway.domain;
+
+import nextstep.subway.dto.LineRequest;
+import nextstep.subway.dto.LineResponse;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+@Entity
+public class Line extends BaseEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(unique = true)
+    private String name;
+
+    private String color;
+
+    private Long upStationId;
+
+    private Long downStationId;
+
+    private Long distance;
```

Comment:
```suggestion
    @ManyToOne(fetch = FetchType.LAZY)
    private Station upStation;

    @ManyToOne(fetch = FetchType.LAZY)
    private Station downStation;

    private Long distance;
```
연관관계를 만들어보는것도 괜찮을거같아요.

그리고 생각해보면 `상행역`, `하행역`, `거리`는 하나의 `구간`이라고 볼수 있고
`노선`은 여러 `구간`으로 이루어져있다고 볼수 있을것같아요! 😀

```suggestion
    @OneToMany(mappedBy = "line")
    private List<Section> sections;
```
이렇게 Section 엔티티로 분리할수도 있겠네요!

다음 단계때 참고정도만 해주세요. 😀